sonar-php-plugin/src/test/java/org/sonar/plugins/php/phpunit/PhpUnitTestResultImporterTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * SonarQube PHP Plugin
+ * Copyright (C) 2010-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.plugins.php.phpunit;
+
+import java.io.File;
+import java.util.HashMap;
+import org.junit.Test;
+import org.sonar.api.batch.fs.InputFile;
+import org.sonar.api.batch.fs.internal.DefaultFileSystem;
+import org.sonar.api.batch.fs.internal.DefaultInputFile;
+import org.sonar.api.batch.sensor.internal.SensorContextTester;
+import org.sonar.api.measures.CoreMetrics;
+import org.sonar.plugins.php.PhpTestUtils;
+import org.sonar.plugins.php.api.Php;
+import org.sonar.test.TestUtils;
+
+public class PhpUnitTestResultImporterTest {
+
+  private PhpUnitTestResultImporter importer;
+
+  private SensorContextTester setUpForSensorContextTester() {
+    SensorContextTester context = SensorContextTester.create(new File("src/test/resources"));
+    importer = new PhpUnitTestResultImporter();
+    return context;
+  }
+
+  @Test()
+  public void shouldGenerateTestsMeasures() {
+    SensorContextTester context = setUpForSensorContextTester();
+    File baseDir = TestUtils.getResource("/org/sonar/plugins/php/phpunit/sensor/src/");
+    DefaultFileSystem fs = new DefaultFileSystem(baseDir);
+    DefaultInputFile appTestFile = new DefaultInputFile("moduleKey", "src/AppTest.php").setType(InputFile.Type.TEST).setLanguage(Php.KEY);
+    DefaultInputFile appSkippedTestFile = new DefaultInputFile("moduleKey", "src/AppSkipTest.php").setType(InputFile.Type.TEST).setLanguage(Php.KEY);
+
+    fs.add(appTestFile);
+    fs.add(appSkippedTestFile);
+    context.setFileSystem(fs);
+
+    String appTest = "moduleKey:" + "src/AppTest.php";
+    String appSkipTest = "moduleKey:" + "src/AppSkipTest.php";
+
+    importer = new PhpUnitTestResultImporter();
+    importer.importReport(TestUtils.getResource(PhpTestUtils.PHPUNIT_REPORT_NAME), context, new HashMap<>());
+
+    PhpTestUtils.assertMeasure(context, appTest, CoreMetrics.TESTS, 1);
```

Comment:
@inverno that's the only place I've found where we assert content of report. But we created only 2 test files so most of report content is ignored. Can we add tests for failures, errors etc?